src/components/ADempiere/Form/VPOS/Order/index.vue

```diff
@@ -307,7 +321,12 @@
                       :is-open="seeConversion"
                     />
                     <el-button slot="reference" type="text" style="color: #000000;font-weight: 604!important;font-size: 100%;" @click="seeConversion = !seeConversion">
-                      {{ formatPrice(currentOrder.grandTotal, pointOfSalesCurrency.iSOCode) }}
+                      {{
```

Comment:
Just can see change for source code with the same functionality, please let it without changes